client/components/select-dropdown/docs/example.jsx

```diff
@@ -125,6 +127,54 @@ const SelectDropdownDemo = React.createClass( {
 					<DropdownItem count={ 3 }>Trashed</DropdownItem>
 				</SelectDropdown>
 
+				<h3 style={ { marginTop: 20 } }>With Icons</h3>
```

Comment:
This example doesn't change selection when something is picked. Would be nice to see the selection change in the example doc.